framework/PageCarousel/PageCarouselNav.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button, Bullseye, ButtonVariant } from '@patternfly/react-core';
+import { AngleLeftIcon, AngleRightIcon } from '@patternfly/react-icons';
+import { CSSProperties, useEffect } from 'react';
+import { useMemo } from 'react';
+
+const pageDotStyle: CSSProperties = {
```

Comment:
This could be converted to styled components to match our pattern elsewhere

Reply:
@keithjgrant I had tried this but couldn't get the dots to render. And the custom styles are not getting applied to the PF Button. The other places that I used it I was not modifying a PF component. Could you look this code over please?

```
const PageNavDotButton = styled(Button)`
  height: 10;
  width: 10;
  border-radius: 50%;
  padding: 0;
  margin-left: 10;
  margin-right: 10;
  ${(props: { isCurrentPage: boolean }) =>
    props.isCurrentPage
      ? css`
          background-color: var(--pf-global--primary-color--100);
        `
      : css`
          background-color: var(--pf-global--disabled-color--200);
        `}
`;

...

  const pageDots = useMemo(() => {
    const dots = [];
    for (let i = 0; i < totalPages; i++) {
      dots.push(
        <PageNavDotButton
          key={i}
          variant={ButtonVariant.plain}
          onClick={() => {
            setPage(i);
          }}
          isCurrentPage={i === currentPage}
        ></PageNavDotButton>
      );
    }
    return dots;
  }, [currentPage, setPage, totalPages]);

```

![image](https://github.com/ansible/ansible-ui/assets/43621546/35ca4901-c813-4aa0-af96-9e8e77ea8295)

Reply:
@keithjgrant Thanks for helping me resolve this! I've updated the PR